internal/static/traces/config_test.go

```diff
@@ -1159,6 +1159,8 @@ remote_write:
       scopes: ["api.metrics"]
       timeout: 2s
 `,
+			//TODO(ptodev): Look into why we need to add a "cipher_suites: []" explicitly.
+			// The expected config should unmarshal it to [], but instead it sets it to nil.
```

Comment:
It seems correct to me that the config would unmarshal it to nil. Else maybe you can explicitly create a list during the convert part (converting from nil to {}string[]). I think that we should avoid adding "cipher_suites: []" explicitly

Reply:
> It seems correct to me that the config would unmarshal it to nil.

I used a debugger in one of the Collector's unit tests, to see if it's `nil` or `[]`. It was `[]`, so I assumed that this is the correct thing to do. I don't know if the unit tests deviates from their prod code path, but I guess it doesn't.

Reply:
I think the way Agent's unit test creates the Collector config isn't the same way the Collector does it. Something is different, but I'm not sure what exactly.